### DISCUSSION TITLE
New release?

### DISCUSSION THREAD

**Discussion by ofacklam** [paullouisageneau/libdatachannel#1255] [General]
Hello @paullouisageneau !
I was just wondering about the rough timeline you had in mind for the next release.
We would be very interested in the new libjuice fixes you introduced.
Thanks
Oliver

**Comment by paullouisageneau** [comment]
I've just released [v0.22.0](https://github.com/paullouisageneau/libdatachannel/releases/tag/v0.22.0). Thank you for your patience!

**Reply by ofacklam** [reply]
Thanks for the heads-up! (and no worries, we were on `master` in the meantime)
Looking forward to testing with 0.22.2
